src/main/resources/com/mathworks/ci/MatlabBuilder/runMatlabTests.m

```diff
@@ -99,6 +122,20 @@
     end
 end
 
+% Produce HTML model coverage report (Not supported below R2018b) 
+if produceModelCoverageHTML
+    if ~exist('sltest.plugins.ModelCoveragePlugin', 'class') || ~htmlModelCoverageSupported
+        warning('MATLAB:testArtifact:cannotGenerateModelCoverageReport', ...
+                'Unable to generate HTML model coverage report. To generate the report, use a Simulink Coverage license with MATLAB R2018b or a newer release.');
```

Comment:
1) Unable to generate a model ...
2) To generate the report -> To generate a report